models/base/fb_ad_creatives.sql

```diff
@@ -1 +1,21 @@
-{{ stitch_fb_ad_creatives() }}
\ No newline at end of file
+{% if var('etl') == 'stitch' %}
+{{ stitch_fb_ad_creatives() }}
+{% elif var('etl') == 'fivetran'%}
+
+with creatives as (
+
+{{ fivetran_fb_ad_creatives() }}
+
+),
+
+utms as (
+
+{{fivetran_fb_url_tag()}}
+
+)
```

Comment:
I would update this to have a `final` cte and then update to use using once you have renamed your `id` in the base table

Reply:
There is a small difference in the location of the `ignore nulls` in the window function.